cmd/meroxa/root/apps/create.go

```diff
@@ -48,7 +48,7 @@ type Create struct {
 
 	flags struct {
 		Path string `long:"path" usage:"path where app was initialized to read its language from app.json (required unless specifying --lang)"`
-		Lang string `long:"lang" short:"l" usage:"language to use (required unless specifying --path)"`
+		Lang string `long:"language" short:"l" usage:"language to use (required unless specifying --path)"`
```

Comment:
I'm not opposed to this change, but was this something requested or something you thought it was worth updating?

Reply:
@janelletavares what docs are you referring to?

Reply:
the command docs.... examples, usage.  the example referred to --language but the flag was actually --lang.  other commands use --langage, so for consistency, chose to change the flag instead of the doc.